bat/lib/mixerlib.bash

```diff
@@ -42,7 +42,7 @@ mixer-create-update() {
 }
 
 mixer-add-rpms() {
-  mkdir -p ./local ./rpms
+  mkdir -p $BATS_TEST_DIRNAME/local-yum $BATS_TEST_DIRNAME/local-rpms
```

Comment:
In the spirit of "This works, but I would write it differently", I note a couple of things.

The `-p` is there to suppress error messages in case the directory exists, you don't need to create any parent directories.

You can use shell expansion to reduce the amount of typing

     mkdir -p $BATS_TEST_DIRNAME/{local-yum,local-rpm}

as the local- is common you could go further, and say `mkdir -p $BATS_TEST_DIRNAME/local-{yum,rpm}` but I wouldn't in a script as it will make it harder to use say `grep` to find where this directory is being made.